File: BikeManagerAPI/BikeManagerAPI/Models/ItemOrcamento.cs
using System;
using System.Collections.Generic;

namespace BikeManagerAPI.Models
{
    public partial class ItemOrcamento
    {
        public int CdProduto { get; set; }
        public int CdOrcamento { get; set; }
        public int CdTabelaPreco { get; set; }
        public decimal VlUnitario { get; set; }
        public decimal VlTotal { get; set; }
        public int QtProduto { get; set; }
        public DateTime DtRegistro { get; set; }
        public DateTime? DtAlteracao { get; set; }

        public Orcamento CdOrcamentoNavigation { get; set; }
        public Produto CdProdutoNavigation { get; set; }
        public TabelaPreco CdTabelaPrecoNavigation { get; set; }
    }
}

File: WebApp/src/app/cadastros/models/itemOrcamento.ts
import { Produto } from "./produto";
import { Orcamento } from "./orcamento";

export class ItemOrcamento {
    cdProduto: number;
    cdOrcamento: number;
    cdTabelaPreco: number;
    vlUnitario: number;
    vlTotal: number;
    qtProduto: number;
    dtRegistro: Date;
    dtAlteracao: Date;
    orcamento: Orcamento;
    produto: Produto;
    //tabelaPreco: TabelaPreco;
}